tests/containers/podman_quadlet.pm

```diff
@@ -67,30 +102,52 @@ _EOF_
     # start the generator and check whether the files are generated
     systemctl("daemon-reload");
     check_unit_states();
-    systemctl("is-active sleeper.service", expect_false => 1);
-    systemctl("is-active sleeper-volume.service", expect_false => 1);
+    for my $unit (@units) {
+        systemctl("is-active $unit", expect_false => 1);
+    }
+
+    # start build unit
+    systemctl("start $unit_name-build.service");
+    record_info('Build output', script_output("journalctl --no-pager -u $unit_name-build"));
+
+    # validate image build
+    systemctl("is-active $unit_name-build.service");
+    validate_script_output('podman images -n', qr/$build_imagetag/);
 
     # start the container
-    systemctl("start sleeper-volume.service");
-    systemctl("start sleeper.service");
+    systemctl("start $unit_name.service");
     check_unit_states();
-    systemctl("is-active sleeper.service");
-    systemctl("is-active sleeper-volume.service");
+    systemctl("is-active $unit_name.service");
+    systemctl("is-active $unit_name-pod.service");
```

Comment:
Quadlet pods only work for newer podman versions, especially in no current SLES version. So the quadlet-pod test should be excluded on pre-SLES 16 versions.

Let me know if you need help with this task.

Reply:
I see... I've seen some checks for the SLE version in other tests, so I'll try to deduce a solution.

Reply:
The `is_sle("<16")` subroutine from `version_utils` should come in handy.

It is in general good to include all tests, and only exclude them in specific situations. So, instead of `is_sle(">16") { /* do the thing */ }`, do `unless(is_sle("<16") { /* do the thing */ }`. Like this, by default it's always executed, on all products and only excluded when specifically needed.

Reply:
Well, apparently `quadlet` is not available in SLE 15.6 's version of `podman` (and supposedly lower versions as well). So I added an inclusion for SLE 16+ in the `lib/main_containers.pm` and didn't bother with disabling the "pod" test if it doesn't run at all.